crates/sui-framework/sources/governance/stake_subsidy.move

```diff
@@ -31,6 +31,16 @@ module sui::stake_subsidy {
         }
     }
 
+    public(friend) fun mint_stake_subsidy_proportional_to_total_stake(
+        subsidy: &mut StakeSubsidy, supply: &mut Supply<SUI>, stake_subsidy_rate: u64, total_stake: u64
+    ) {
+        let amount_to_mint = (total_stake as u128) * (stake_subsidy_rate as u128) / BASIS_POINT_DENOMINATOR;
```

Comment:
Where does the subsidy come from?

Reply:
Please rename this function by appending _testnet

Reply:
Although it has the same semantics, I think it's good to do
```suggestion
        let amount_to_mint = ((total_stake as u128) * (stake_subsidy_rate as u128)) / BASIS_POINT_DENOMINATOR;
```

to make it clear that multiplication before division is important to minimize rounding error.